From: "Zenghui Yu (Huawei)" <zenghui.yu@linux.dev>
Date: Fri, 7 Aug 2026 01:07:01 +0800
Subject: [PATCH] mm/damon/core: remove declaration of __damon_commit_ctx()

It was added by commit b1471afe4d10 ("mm/damon/core: do parameter testing
commit on damon_start()") but is actually not needed.  Remove it.

Changes from v1
- v1: https://lore.kernel.org/20260806170701.79266-1-zenghui.yu@linux.dev
- Collect R-b: from SJ.
- Rebase to latest mm-new.

Cc: "Zenghui Yu (Huawei)" <zenghui.yu@linux.dev>
Signed-off-by: Zenghui Yu (Huawei) <zenghui.yu@linux.dev>
Reviewed-by: SJ Park <sj@kernel.org>
Signed-off-by: SJ Park <sj@kernel.org>
---
 mm/damon/core.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/mm/damon/core.c b/mm/damon/core.c
index b3f8bea774c2d..79515ef03fc2a 100644
--- a/mm/damon/core.c
+++ b/mm/damon/core.c
@@ -1961,8 +1961,6 @@ static int __damon_start(struct damon_ctx *ctx)
 	return err;
 }
 
-static int __damon_commit_ctx(struct damon_ctx *dst, struct damon_ctx *src);
-
 /**
  * damon_start() - Starts the monitorings for a given group of contexts.
  * @ctxs:	an array of the pointers for contexts to start monitoring
-- 
2.47.3

